tests/crypto/host/CMakeLists.txt

```diff
@@ -21,134 +20,8 @@ add_executable(hostcrypto
     ../tests.c
     ../utils.c)
 
-
-# ========================= TestCRL & TestASN1 ================================
-
-add_custom_command(TARGET hostcrypto
```

Comment:
Just curious, what was the reasoning of replacing this with a Bash script?

Reply:
Many of the `COMMAND` lines are linux commands (`sleep`, `awk`, `tail`, etc.) which cmake runs in CMD and just fail. Encapsulating them in a bash script allows a clearer specification of the execution environment instead of trying to branch individual commands for cross-platform compat at the cmake level.